src/objects/Pieces/Square.js

```diff
@@ -14,9 +14,6 @@ export default class Square extends Base {
     ];
   }
 
-  get rotate() {
-  }
-
-  get unrotate() {
+  rotate(_direction) {
```

Comment:
I know. This is overriding the one of the superclass.
We don't want rotations to do anything at all on Square pieces, otherwise, due to way calculations are being done, they would shift 1 square to the side every 2 rotations